packages/headless/src/api/service/insight/user-actions/user-actions-request.ts

```diff
@@ -45,10 +40,8 @@ export const buildInsightUserActionsRequest = (
     ...baseInsightRequest(req, 'POST', 'application/json', '/useractions'),
     requestParams: {
       ticketCreationDate: params.ticketCreationDate,
-      numberSessionsBefore: params.numberSessionsBefore ?? 50,
-      numberSessionsAfter: params.numberSessionsAfter ?? 50,
-      maximumSessionInactivityMinutes:
```

Comment:
Why did you remove this parameter?

Reply:
We found out more about snowflake and how visits are determined. The data is already segmented by visitID which helps us a lot to determine sessions but "visits" are defined as 30 minutes of inactivity by the data-platform and we can't change that. So trying to work around an arbitrary value for `maximumSessionInactivityMinutes` seems like a total nightmare and I suggested we remove it.

Reply:
Thanks for the explanation. I totally agree with you :+1: